Assets/Code/src/Runtime/Match/Components/MatchPauseController.cs

```diff
@@ -25,9 +25,9 @@ public class MatchPauseController : MonoBehaviour {
   /// </summary>
   void Update() {
     if (MatchManager == null || !MatchManager.IsLocal) return;
-    if (MatchManager.IsPaused) {
+    if (MatchManager.MatchController.CurrentState.StateID == MatchStateID.Pause) {
```

Comment:
MatchController.CurrentState.StateID is being called a lot, might be better if there was property that made it easier to access.